### DISCUSSION TITLE
Postal web page not working

### DISCUSSION THREAD

**Discussion by PCL-CrisKolkman** [postalserver/postal#1921] [Help with using Postal]
Hello,

I've setup a complete new VM (Ubuntu 18.04 LTS) and followed every step in:
https://docs.postalserver.io/install/prerequisites and https://docs.postalserver.io/install/installation

When I want to go to the web page, I get:
ERR_SSL_PROTOCOL_ERROR

I probably missed something in the configuration of Caddy but that's not documented on above pages as well, so I have no idea what could be wrong.

**Comment by johnceci-m3** [comment]
add a second entry in your Caddyfile that looks something like this, it will allow you to access your admin over port 80, obviously you should have an SSL certificate, but if you want to get it running...

you.domain.com:80 {
    reverse_proxy 127.0.0.1:5000
}

**Reply by PCL-CrisKolkman** [reply]
> add a second entry in your Caddyfile that looks something like this, it will allow you to access your admin over port 80, obviously you should have an SSL certificate, but if you want to get it running...
> 
> you.domain.com:80 { reverse_proxy 127.0.0.1:5000 }

Hello @johnceci-m3,

Thank you for your reply.

This is in the documentation I've linked to:

![image](https://user-images.githubusercontent.com/97805195/159264183-4a47c297-7534-42b6-b2c4-d1bb3f66cef3.png)

I kinda thought this would issue an certificate (like letsencrypt?).
